modules/ROOT/pages/cloudhub-connector.adoc

```diff
@@ -41,34 +37,34 @@ image::cloudhub-connector-props-blank.png[]
 +
-image::cloudhub-connector-a71a6.png[cloudhub-basic-auth-config]
+image::cloudhub-connector-a71a6.png[]
 +
 . Make sure that you have selected the *Create notification* operation and set the properties for:
-.. *Domain*: This should be the domain you set for your application when you first created it following the xref:general:getting-started:deploy-to-cloudhub.adoc[Deploy to CloudHub tutorial]. Note that this is not the complete domain name (myapp.cloudhub.io) but only the application name (myapp)
-.. *Message*: The message you want your notification to show (e.g. "_Your application has been hit_"). In the example below, the notification will simply display a "_Hello World!_" message.
-.. *Priority*: The type of the notification. By selecting "INFO", your notification will display an informative message while selecting "ERROR" will display a red alert message.
-
+.. *Domain*: This should be the domain you set for your application when you first created it following the xref:general:getting-started:deploy-to-cloudhub.adoc[Deploy to CloudHub tutorial]. Note that this is not the complete domain name (myapp.cloudhub.io) but only the application name (myapp).
+.. *Message*: The message you want your notification to show (for example, "Your application has been hit"). In the example below, the notification will simply display a "Hello World!" message.
+.. *Priority*: The type of the notification. By selecting "INFO", your notification displays an informative message while selecting "ERROR" displays a red alert message.
 +
 image::cloudhub-connector-settings.png[]
 +
 . Additional information can be included with the notification by configuring the "Custom Properties" section.
++
 [NOTE]
-The expected "Custom Properties" format is `Map <String, String>` and is typically referenced using a bean or a variable where you have defined that map, for example, `\#[flowVars.magicMap]`,
+The expected "Custom Properties" format is `Map <String, String>` and is typically referenced using a bean or a variable where you have defined that map, for example, `#[flowVars.magicMap]`,
 or by defining a map using MEL `#[['key1':'value1']]`
-
++
 . At this point, your application is set to be deployed and run. When your domain root is visited, the *Create notification* operation in the CloudHub connector creates a "Hello World!" notification message for the domain that the application is currently running on.
 
 [IMPORTANT]
-If you always want to create notifications on CloudHub, even when running locally, you can use the *Create notification* operation. This requires you to configure the connector with your username and password.
+If you always want to create notifications in CloudHub, even when running locally, you can use the `Create notification` operation. This requires you to configure the connector with your Anypoint Platform username and password.
 
-NOTE]
+[NOTE]
 The "Retrieve Logs" operation is no longer supported - to download logs use CH API or Anypoint CLI.
```

Comment:
Not clear why Retrieve Logs operation is in quotes. Recommend removing them. It's capitalized, so that should be sufficient. If the name of the operation is in the GUI, it should be bold.

Reply:
Re "supported - to download" 
I recommend changing less common punctuation like em-dashes and en-dashes to something more common.